netbout-web/src/main/java/com/netbout/rest/TkApp.java

```diff
@@ -107,16 +121,23 @@ private static Take make(final Base base) throws IOException {
                 )
             );
         }
+        TkAppAuth appAuth;
```

Comment:
@antonini let's rename this to a single word: `auth` is good enough.
See this article http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html